index.html

```diff
@@ -10,9 +10,58 @@
 </head>
 
 <body class="h-100">
+    <nav class="navbar navbar-light bg-light mb-5">
+      <a class="navbar-brand" href="index.html">PLACEHOLDER TITLE HERE</a>
+    </nav>
+  <div class="container">
+
+    <!-- Modal with instructions -->
+    <div class="modal fade" id="instructionModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="false">
+      <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title" id="exampleModalLongTitle">How to use this webpage!</h5>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body">
+              Instruction blurb should go here
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- nav bar - clicking on returns you to index page -->
+  
+
+  <!-- This div will handle the image uploading -->
+  <div class="jumbotron jumbotron-fluid">
+    <div class="container">
+      <h1 class="display-4">Place your image here!</h1>
+      <form>
+          <div class="form-group">
+            <label for="exampleInputEmail1">Image URL</label>
+            <input type="url" class="form-control" id="imgUrlInput" aria-describedby="emailHelp" placeholder="Enter email">
+          </div>
+          <button type="submit" class="btn btn-primary">Submit</button>
+        </form>
+    </div>
+  </div>
+
+  <!-- This is a placeholder div for the image carousel -->
+  <div id="imgCaro">
```

Comment:
maybe `id="imgCarousel"` instead of `id="imgCaro"`. Took a min before my brain settled on `Caro = Carousel`